### PR TITLE
Added isolation to simulation

### PR DESCRIPTION
**Pull request by joyglitch**
Isolation for mildly infected agents

### CODE REVIEW

File: src/isolationcompartment.cpp [hunk 1]
```diff
@@ -12,5 +12,20 @@
...
 }
+
+vector<Agent *> IsolationCompartment::isolatedAgents() {
+    return isolated;
+}
+
+void IsolationCompartment::ReleaseRecoveredAgents(int index) {
+    if(index < 0 || index >= (int)isolated.size());
+    isolated.erase(isolated.begin(), isolated.begin() + index);
+    //Should this function return the agent to be removed from the isolated vector?
```
Comment (danyal1234) [inline src/isolationcompartment.cpp:25]: yeah 100%, we dont wanna lost track of them

Comment (joyglitch) [reply src/isolationcompartment.cpp:25]: Alright going to fix

File: src/main.cpp [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 #include <iostream>
 #include <time.h>
+// should be the only thing included after finished
```
Comment (danyal1234) [inline src/main.cpp]: remove this

Comment (joyglitch) [reply src/main.cpp]: That's my bad

**Pull request merged**